docs/usage.rst

```diff
@@ -11,34 +11,22 @@ Ansible Builder can execute two separate steps. The first step is to create a bu
 The ``build`` command
 ---------------------
 
-The ``ansible-builder build`` command takes an execution environment definition file as an input. It outputs a build instruction file (Containerfile for Podman, Dockerfile for Docker), creates the build context necessary for building an execution environment image, and builds that image. The image can be re-built with the build context elsewhere, and give the same result. By default, it looks for a file named ``execution-environment.yml`` in the current directory.
+The ``ansible-builder build`` command:
 
-This example uses the following ``execution-environment.yml``
-file as a starting point:
+* takes an :ref:`execution environment definition file<builder_ee_definition>` as an input,
+* outputs a build instruction file (Containerfile for Podman, Dockerfile for Docker),
+* creates a build context necessary for building an execution environment image,
+* builds the image.
 
-.. code:: yaml
-
-    ---
-    version: 3
-    dependencies:
-      galaxy: requirements.yml
-
-
-The content of ``requirements.yml``:
-
-.. code:: yaml
-
-   ---
-   collections:
-     - name: awx.awx
+By default, it looks for a file named ``execution-environment.yml`` in the current directory.
 
-To build an execution environment using the files above, run:
+To build an execution environment using the default definition file, run:
 
 .. code::
 
    $ ansible-builder build
    ...
-   STEP 7: COMMIT my-awx-ee
+   STEP 7: COMMIT my-ee
```

Comment:
Technically, this output isn't correct, especially the container tag name, which is not the default. Unless the `-v` verbosity option is used, the only thing the user will see with this command is (with slight modification to context path):

```
Running command:
  podman build -f context/Containerfile -t ansible-execution-env:latest context
Complete! The build context can be found at: /path/to/context
```

Reply:
> so can i just update the output showing this ^ ?

Sure

Reply:
thanks, done, PTAL